vnext/ReactUWP/Views/ViewViewManager.h

```diff
@@ -3,12 +3,12 @@
 
 #pragma once
 
+#include <Views/ViewPanel.h>
 #include <Views/FrameworkElementViewManager.h>
```

Comment:
Why is this needed now? It's still just a reference, right?

Reply:
TryUpdateView needs it.